client/state/data-layer/wpcom/sites/activity/index.js

```diff
@@ -72,6 +72,8 @@ export const receiveActivityLogError = ( { dispatch }, { siteId }, error ) => {
 
 export default {
 	[ ACTIVITY_LOG_REQUEST ]: [
-		dispatchRequest( handleActivityLogRequest, receiveActivityLog, receiveActivityLogError ),
+		dispatchRequest( handleActivityLogRequest, receiveActivityLog, receiveActivityLogError, {
```

Comment:
I know this is existing code, but I like to encourage us to use descriptive names like `fetchActivityLog` or `updateActivityLog` etc…